completion_aggregator/api/v1/views.py

```diff
@@ -395,7 +399,10 @@ def _parse_aggregator(self, course_key, params=None):
 
         root_block = params.get('root_block')
         if root_block:
-            root_block = UsageKey.from_string(root_block).map_into_course(course_key)
+            try:
+                root_block = UsageKey.from_string(root_block).map_into_course(course_key)
+            except InvalidKeyError:
+                raise NotFound
```

Comment:
Could we please make these messages more verbose? E.g.:
```python
raise NotFound("Invalid block key: '{}'.".format(root_block))
```

Reply:
Done also to the other places I added checks.